pkg/koordlet/resourceexecutor/updater.go

```diff
@@ -152,6 +156,36 @@ func (u *CgroupResourceUpdater) SetUpdateFunc(updateFunc UpdateFunc, mergeUpdate
 	u.mergeUpdateFunc = mergeUpdateFunc
 }
 
+// CgroupFileWriteIfDifferent writes the cgroup file if current value is different from the given value.
+func (u *CgroupResourceUpdater) cgroupFileWriteIfDifferent() (bool, error) {
```

Comment:
The naming `(*CgroupResourceUpdater).cgroupFileWriteIfDifferent` is wordy. And should all CgroupResourceUpdater implement this method? How about moving the invoke of auditor into the update functions?